enigma-crypto/src/symmetric.rs

```diff
@@ -74,6 +74,12 @@ pub fn decrypt(cipheriv: &[u8], key: &SymmetricKey) -> Result<Vec<u8>, CryptoErr
     Ok(decrypted_data.to_vec())
 }
 
+/// Get tag size
+pub fn get_tag_size() -> usize {
+    AES_MODE.tag_len()
+}
```

Comment:
please add an `#[inline(always)]` annotation to this, to reduce the runtime cost. `AES_MODE.tag_len()` (which is really `ring::aead::Algorithm::tag_len()`) is also defined with this, because all it does is return a constant from inside the `ring::aead` scope.

I understand the maintainability consideration in moving this value out of eng_wasm and into the runtime, but is the runtime overhead of context-switching from WASM to Rust really worth all this trouble?

(also, how is this relevant to the main task? I must be missing something)

Reply:
@reuvenpo I understand your concern about runtime overhead, I implemented this for maintainability, but never was sure that this worth the overhead. Will be happy to discuss this f2f.
I don't understand your comment about "main task". What is main task?

Reply:
As we decided, I define the constants in `crypto_wasm.rs` to avoid runtime overhead